### PR TITLE
Add Doc: Configure S2I and B2I Webhooks

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@yunify.com>

### CODE REVIEW

**Review by LinuxSuRen — APPROVED** [review]
/lgtm

File: content/en/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+- You need to create a workspace, a project (`demo-project`) and an account (`project-regular`). The account must be invited to the project with the role of `operator`. For more information, see [Create Workspaces, Projects, Accounts and Roles](../../../quick-start/create-workspace-and-project/).
+- You need to create an S2I Image Builder and a B2I Image Builder. For more information, refer to [Source to Image: Publish an App without a Dockerfile](../source-to-image/) and [Binary to Image: Publish an Artifact to Kubernetes](../binary-to-image/).
+
+## Configure an S2I Webhook
+
+### Step 1: Expose the S2I trigger Service
+
+1. Log in to the KubeSphere web console as `admin`. Click **Platform** in the top left corner and then select **Cluster Management**.
+
+2. In **Services** under **Application Workloads**, select **kubesphere-devops-system** from the drop-down list and click **s2ioperator-trigger-service** to go to its details page.
```
Comment (Sherlock113) [inline content/en/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks.md:25]: its details page.
->
its detail page.

PS: I think both of them are correct as I found both versions in Google. Please use the singular form for consistency. Meaning-wise, I prefer to use `detail` as you are actually talking about one service here. Please change this across the guide.

File: content/en/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+### Step 2: Configure an S2I webhook
+
+1. Log out of KubeSphere and log back in as `project-regular`. Go to `demo-project`.
+
+2. In **Image Builder**, click the S2I Image Builder to go to its details page.
+
+   ![click-s2i](/images/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks/click-s2i.png)
+
+3. You can see an auto-generated link shown in **Remote Trigger Link**. Copy the contents of `/s2itrigger/v1alpha1/general/namespaces/demo-project/s2ibuilders/felixnoo-s2i-sample-latest-zhd/` as it will be included in the S2I webhook URL.
```
Comment (Sherlock113) [inline content/en/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks.md:55]: Delete `the contents of`.

File: content/en/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+   ![click-add-webhook](/images/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks/click-add-webhook.png)
+
+5. In **Payload URL**, enter `http://<IP>:<Service NodePort>/s2itrigger/v1alpha1/general/namespaces/demo-project/s2ibuilders/felixnoo-s2i-sample-latest-zhd/`. You can select trigger events based on your needs and then click **Add webhook**. This tutorial chooses **Just the push event** for demonstration purposes.
+
+   ![add-payload-url](/images/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks/add-payload-url.png)
+
+   {{< notice note >}}
+
+   `<IP>` is your own IP address, `<Service NodePort>` is the NodePort you get in step 1, and `/s2itrigger/v1alpha1/general/namespaces/demo-project/s2ibuilders/felixnoo-s2i-sample-latest-zhd/` is from the S2I remote trigger link. Make sure you use your own IP, Service NodePort and S2I remote trigger link. You may also need to configure necessary port forwarding rules and open port in your security groups depending on where your Kubernetes clusters are deployed.
```
Comment (Sherlock113) [inline content/en/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks.md:69]: open port
->
open the port

If you want to mean a specific port, omit the article. Here are some examples for you:

image
image
image
image

Comment (Sherlock113) [inline content/en/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks.md:69]: depending on where your Kubernetes clusters are deployed.
->
depending on where your Kubernetes cluster is deployed.

File: content/en/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+   ![b2i-trigger-link](/images/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks/b2i-trigger-link.png)
+
+3. Add the payload URL in the source code repository. The B2I payload URL format is the same as that of S2I payload URL.
+
+   ![b2i-payload-url](/images/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks/b2i-payload-url.png)
+
+   {{< notice note >}}
+
+   You may need to configure necessary port forwarding rules and open port in your security groups depending on where your Kubernetes clusters are deployed.
```
Comment (Sherlock113) [inline content/en/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks.md:97]: open port in your security groups depending on where your Kubernetes clusters are deployed.
->
open the port in your security groups depending on where your Kubernetes cluster is deployed.

File: content/en/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+   ![webhook-delivery](/images/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks/webhook-delivery.png)
+
+7. After you finish all the above operations, the S2I Image Builder will be automatically triggered if there is a push event to the source code repository.
+
+   ![s2i-auto-build](/images/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks/s2i-auto-build.png)
+
+## Configure a B2I Webhook
+
+The method of configuring B2I webhook is the same as configuring S2I webhook.
```
Comment (Sherlock113) [inline content/en/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks.md:83]: The method of configuring B2I webhook is the same as configuring S2I webhook.
->
You can follow the same steps to configure a B2I webhook.

**Pull request merged**